React/wbooks/src/Navbar/index.js

```diff
@@ -0,0 +1,39 @@
+import React, { Component } from 'react';
+import { withRouter } from 'react-router-dom';
+import { connect } from 'react-redux';
+
+import authActions from '../redux/auth/actions';
+
+import NavBar from './layout';
+
+class NavbarContainer extends Component {
+  state = { isMenuDisplayed: false, areNotificationsDisplayed: false };
+
+  handleToogleMenu = () => {
+    this.setState({ isMenuDisplayed: !this.state.isMenuDisplayed });
+  };
+
+  handleToogleNotif = () => {
+    this.setState({ areNotificationsDisplayed: !this.state.areNotificationsDisplayed });
+  };
+
+  logoutHandler = () => {
+    this.props.dispatch(authActions.logout());
+  };
+
+  render() {
+    return (
+      <NavBar
+        logoutHandler={this.logoutHandler}
+        isMenuDisplayed={this.state.isMenuDisplayed}
+        areNotificationsDisplayed={this.state.areNotificationsDisplayed}
+        onToogleNotif={this.handleToogleNotif}
+        onToogleMenu={this.handleToogleMenu}
+      />
+    );
+  }
+}
+
+const mapStateToProps = state => ({ logState: state.auth.logState });
+
+export default withRouter(connect(mapStateToProps)(NavbarContainer));
```

Comment:
I don't think there's need to wrap this with withRouter. You aren't using the props the withRouter HOC gives you.